.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Ignore WebConfig because it contains sensitive values
-Web.config
+Authy2FA/Web.config
```

Comment:
ok.  If we're doing this just to avoid having secrets in the web.config as we develop a solution that I have been prefering is leveraging the `file` attribute of the `appSettings` element as shown here:

http://peterkellner.net/2008/02/23/webconfigbestpractice/

I create a file call Local.config and tell git to ignore that file.  If its there (during development) ASP.NET will use those values, if not it will fall back to whatever is in the web.config.

It does not work with connectionstrings, but I think for the sames that should not matter.